docs/static/images/users/nasa.svg

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What do you think of using https://jakearchibald.github.io/svgomg/ to compress a bit the svg?